java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/calendar/ArrowFlightJdbcTimeStampVectorAccessorTest.java

```diff
@@ -176,13 +176,12 @@ public void testShouldGetTimestampReturnValidTimestampWithCalendar() throws Exce
     TimeZone timeZoneForVector = getTimeZoneForVector(vector);
 
     accessorIterator.iterate(vector, (accessor, currentRow) -> {
-      final Timestamp resultWithoutCalendar = accessor.getTimestamp(null);
       final Timestamp result = accessor.getTimestamp(calendar);
 
-      long offset = timeZone.getOffset(resultWithoutCalendar.getTime()) -
-          timeZoneForVector.getOffset(resultWithoutCalendar.getTime());
+      long offset = timeZone.getOffset(result.getTime()) -
+          timeZoneForVector.getOffset(result.getTime());
 
-      collector.checkThat(resultWithoutCalendar.getTime() - result.getTime(), is(offset));
+      collector.checkThat(result.getTime(), is(getTimestampForVector(currentRow).getTime() - offset ));
```

Comment:
```suggestion
      collector.checkThat(result.getTime(), is(getTimestampForVector(currentRow).getTime() - offset));
```
Lets just be sure that checkstyle doesn't complain about anything (run `mvn install` without additional flags)